airflow/post_install.sh

```diff
@@ -14,7 +14,9 @@ python $AIRFLOW_HOME/utils/set_default_variables.py \
     source_dataset="{{ var.value.PIPELINE_DATASET }}" \
     source_table="messages_scored_" \
     bigquery_segment_vessel="segment_vessel" \
+    bigquery_segment_info="segment_info" \
     bigquery_tracks="tracks" \
+    bigquery_vessel_info="vessel_info" \
```

Comment:
It doesn't break since it has an empty line after all of this. The idea here is that you can reorder these lines or add new lines without needing to change the last line, so that it doesn't show up in the diff. I've been doing this in all my pipelines so far.